Source/Api/Impl/IdentityClientImpl.swift

```diff
@@ -17,44 +17,32 @@ import Foundation
 /**
 - Author: Allan Melo
 */
-class IdentityClientImpl: IdentityClient {
+class IdentityClientImpl {
```

Comment:
We can rename this to IdentityClient, isn't it?

Reply:
Yeah Victor, now we can change it, in first moment we needed to keep the same name from Java API 👎 . But now we are free to change. I will do that. Thx